File: ENTORNOS/PYTHON ALTERNATIVAS/Ejercicio9.py
#Gonzalo Carretero
#Algoritmo que pida tres números y los muestre ordenados (de mayor a menor)

print("Introduzca tres numeros")
num1 = int(input())
num2 = int(input())
num3 = int(input())

if num1 > num2:
    if num2 > num3:
        print("El mayor es",num1,"el segundo mayor es",num2,"y el ultimo es",num3,) 
    else:
        print("El mayor es",num1,"el segundo mayor es",num3,"y el ultimo es",num2,)
else:
    if num2 > num1:
        if num1 > num3:
            print("El mayor es",num2,"el segundo mayor es",num1,"y el ultimo es",num3,)
        else:
            print("El mayor es",num2,"el segundo mayor es",num3,"y el ultimo es",num1,)
    if num3 > num1:
        print("El mayor es",num3,"el segundo mayor es",num2,"y el ultimo es",num1,)
    else:
        print("El mayor es",num3,"el segundo mayor es",num1,"y el ultimo es",num2,)

File: PROGRAMACION/TEMA 4/RELACION A/NIVEL JEDI/Ejercicio9/src/ejercicio9/Ejercicio9.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Main.java to edit this template
 */
package ejercicio9;

import java.util.Scanner;

/*
Realiza un programa que pida introducir tres valores enteros y nos diga cuál 
de ellos es el más elevado. Impleméntalo creando únicamente una función a la 
que le pasemos dos valores (no tres) y nos devuelva el máximo de los dos valores
*/
/**
 *
 * @author usuario
 */
public class Ejercicio9 {

    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
        // TODO code application logic here
        
        Scanner sc = new Scanner(System.in);
        
        int num1 , num2 , num3;
        
        
        System.out.println("Introduzca un numero: ");
        num1 = sc.nextInt();
        
        System.out.println("Introduzca otro numero: ");
        num2 = sc.nextInt();
        
        System.out.println("Introduzca otro numero: ");
        num3 = sc.nextInt();
        
        num1 = valores(num1,num2);
        
        num2 = num3;
        
        System.out.println("Valor maximo: "+valores(num1,num2));
        
    }
    
    public static int valores (int num1 ,int  num2){
        
        int mayor;   
        
        if (num1 < num2) {
            mayor = num2;
        }
        else{
            mayor = num1;
        }
        
        
        return mayor;
        
    }
    
}
